scripts/check-linters

```diff
@@ -23,6 +23,11 @@ for i in $filelist; do
     if [[ $i =~ ^3rdparty/ ]]; then
         continue;
     fi
+    # Temporary, to remove later
```

Comment:
The linter forced quotes around bash variables, which caused the compiler flags to not be applied correctly. A subsequent PR will be created to remove the bypass once this PR is complete.